frontend/src/components/ParticipantsSearch.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-combobox
+    clearable
+    label="Search participants"
+    :items="sortedParticipants"
+    item-text="github"
```

Comment:
I know that you replicated the previous solution here but it would make user's life much easier if we could render/display results for each item in the format:
"`first_name` `last_name` (`slack`/`github`)"

In the documentation I found that combobox accepts stings:
https://vuetifyjs.com/en/api/v-combobox/#props
Not sure if that applies to the example above. 

Yet, given that we will use this component for different purposes and there are three different sources of information about who selectedParticipant is it would be great to have all that information on the list not to wonder about if the person with slack nick A is also the person with github nick B (and they do vary)